ibtk/src/utilities/libmesh_utilities.cpp

```diff
@@ -301,7 +301,6 @@ std::vector<libMeshWrappers::BoundingBox>
 get_local_active_element_bounding_boxes(const libMesh::MeshBase& mesh, const libMesh::System& X_system)
 {
     const unsigned int dim = mesh.mesh_dimension();
```

Comment:
@AminKolah here's at least one bug with the surface mesh code: `mesh_dimension()` is the topological dimension but for bounding boxes we want the spatial dimension. I'll fix this.